pages/[id]/BuildConfig.js

```diff
@@ -0,0 +1,89 @@
+import { useState } from 'react';
+import Page from '../../components/page';
+import Card, {
+    CardHeader,
+    CardTitle,
+    CardContent,
+  } from '../../components/card';
+import { HACKATHON_NAVBAR } from '../../constants';
+import {
+    db,
+    getHackathonPaths,
+    getHackathons,
+  } from '../../utility/firebase';
+  
+export default function BuildConfig({id, hackathons}) {
+    const [buildConfig, setBuildConfig] = useState({});
+
+    db.collection('Hackathons').doc(id).onSnapshot(doc => {
```

Comment:
i think there's a util for this in utility\firebase.js

Reply:
oh yea are you referring to getDocument? i was gonna use it, however I think i had permission issues and Ian told me not to use it lol and to just use db.collection.doc